lib/asset/blogArticle.js

```diff
@@ -13,11 +13,57 @@ const { ensureCheerio, fixLinkTargets } = require('./util');
 
 /** */
 class BlogArticle extends BaseAsset {
-    constructor() {
-        super();
+    get ALLOWED_METADATA() {
```

Comment:
Same here, `static get`? (and `BaseAsset.ALLOWED_METADATA` below)

Reply:
I will move to `static get`. Please note, after running `eslint` I moved the `if {} else if {}` to `switch/case` statements.

Reply:
@ptomato now I remember why I didn't use `static get` for the allowed metadata. The children assets expand the metadata allowed. So the `_check_metadata_is_allowed` uses `this.ALLOWED_METADATA` which is different depending on the asset. If I use `static get` I will have to reimplement `_check_metadata_is_allowed` in each asset.